### PR TITLE
[math-ci] changed CI dependencies (#908)

### PR DESCRIPTION
**Pull request by amd-garydeng**
Summary of proposed changes:
-dependency list change to support newer builds of rocBLAS

### CODE REVIEW

**Review by daineAMD — APPROVED** [review]

**Comment by TorreZuk** [comment]
@amd-garydeng or @daineAMD should merge soon.

**Review by TorreZuk — CHANGES_REQUESTED** [review]
@amd-garydeng you messed up this PR with the develop merge.  You now want to remove the tip commit like git reset --hard HEAD^

**Comment by TorreZuk** [comment]
> @amd-garydeng you messed up this PR with the develop merge. You now want to remove the tip commit like git reset --hard HEAD^

Or just start again with 6.3 and cherry pick if too messed up

**Review by TorreZuk — APPROVED** [review]

**Comment by amd-garydeng** [comment]
@TorreZuk Just fixed it. Not sure why I merged develop into this.

File: .jenkins/static.groovy [hunk 1]
```diff
@@ -18,11 +18,10 @@ def runCI =
 
     if (env.BRANCH_NAME ==~ /PR-\d+/ && pullRequest.labels.contains("noSolver"))
     {
-        prj.libraryDependencies = ['rocBLAS', 'hipBLAS-common']
-    }
+        prj.libraryDependencies = ['hipBLAS-common', 'hipBLASLt', 'rocBLAS']    }
```
Comment (daineAMD) [inline .jenkins/static.groovy:21]: weird formatting here

**Review by daineAMD — APPROVED** [review]

**Comment by TorreZuk** [comment]
@amd-garydeng this looks good to merge with or without bracket formatting fix. It will unblock my PR CI

**Pull request merged**